doc/articles/using-uno-ui.md

```diff
@@ -369,6 +369,24 @@ of `IImageSourceDownloader`, which provides a localy downloaded representation o
 On Android, for a more efficient handling of images, the Image control can be provided a 
```

Comment:
This should be updated, it's not a question of 'more efficient,' external urls don't work on Android unless UniversalImageLoader is installed.